deb/ar.go

```diff
@@ -144,17 +147,22 @@ func parseArEntry(line []byte) (*ArEntry, error) {
 		FileMode: strings.TrimSpace(string(line[48:58])),
 	}
 
-	for target, value := range map[*int64][]byte{
-		&entry.Timestamp: line[16:28],
-		&entry.OwnerID:   line[28:34],
-		&entry.GroupID:   line[34:40],
-		&entry.Size:      line[48:58],
+	for target, value := range map[entryField][]byte{
+		entryField{"Timestamp", &entry.Timestamp}: line[16:28],
+		entryField{"OwnerID", &entry.OwnerID}:     line[28:34],
+		entryField{"GroupID", &entry.GroupID}:     line[34:40],
+		entryField{"Size", &entry.Size}:           line[48:58],
```

Comment:
I added this struct because it makes errors clearer and debugging easier. If this isn't desirable, I can remove it.